test/source/mock/google/google-endpoints.ts

```diff
@@ -81,55 +120,28 @@ export const getMockGoogleEndpoints = (oauth: OauthMock): HandlersDefinition =>
       throw new Error(`Method not implemented for ${req.url}: ${req.method}`);
     },
     '/v1/people:searchContacts': async ({ query: { query } }, req) => {
-      if (!isGet(req)) {
+      if (!isGet(req) || !config) {
         throw new HttpClientErr(`Method not implemented for ${req.url}: ${req.method}`);
       }
-      const empty = {};
-      const acct = oauth.checkAuthorizationHeaderWithAccessToken(req.headers.authorization);
-      if (acct === 'ci.tests.gmail@flowcrypt.test') {
-        if (query === 'contact') {
-          return {
-            results: [{ person: { emailAddresses: [{ metadata: { primary: true }, value: 'contact.test@flowcrypt.com' }] } }],
-          };
-        } else if (query === 'testsearchorder') {
-          return {
-            results: [
-              { person: { emailAddresses: [{ metadata: { primary: true }, value: 'testsearchorder1@flowcrypt.com' }] } },
-              { person: { emailAddresses: [{ metadata: { primary: true }, value: 'testsearchorder2@flowcrypt.com' }] } },
-              { person: { emailAddresses: [{ metadata: { primary: true }, value: 'testsearchorder3@flowcrypt.com' }] } },
-              { person: { emailAddresses: [{ metadata: { primary: true }, value: 'testsearchorder4@flowcrypt.com' }] } },
-              { person: { emailAddresses: [{ metadata: { primary: true }, value: 'testsearchorder5@flowcrypt.com' }] } },
-              { person: { emailAddresses: [{ metadata: { primary: true }, value: 'testsearchorder6@flowcrypt.com' }] } },
-            ],
-          };
-        } else {
-          return empty;
-        }
-      } else {
-        return empty;
+      if (!config.contacts) {
+        return {};
```

Comment:
What if we'll return `{}` even when config isn't set? (and for `/othercontacts` too)
Then we won't need to declare empty config like we have in some tests `{ google: { contacts: [] } }`